js/app/src/components/StatusTracker/ToastContent.svelte

```diff
@@ -0,0 +1,173 @@
+<script lang="ts">
+	import { Alert, Information, Warning } from "@gradio/icons";
+	import { createEventDispatcher, onMount, afterUpdate } from "svelte";
+	import { fade } from "svelte/transition";
+	import type { ThemeMode } from "../types";
+
+	export let message: string = "";
+	export let description: string = "";
+	export let type: string = "";
+	export let theme_color: string = "";
+	export let theme_mode: ThemeMode;
+	export let id: number;
+
+	const dispatch = createEventDispatcher();
+
+	function close_message() {
+		dispatch("close", id);
+	}
+
+	let lightStyles = {
```

Comment:
Actually, I think the better approach might be to keep our CSS in the CSS instead of putting it in javascript. So what I would suggest is, instead of have theme_color as a prop as well as theme_mode, we only take in the `type` and add that as a class and everything in the CSS responds to that. So the div below would become:
```
<div class="toast-body {type}">
```
and then the CSS at the bottom becomes
```
<style>
  .toast-body.error {
    background-color: var(--color-red-50);
    color: var(--color-red-700);
    ...
  }
  :global(.dark) .toast-body.error {
    background-color: var(--color-red-950);
    ...
  }

Reply:
Since we only have 3 different types, I think this is the most manageable approach.